src/Client/Client.php

```diff
@@ -17,11 +17,12 @@ class Client
     private ClientInterface $client;
 
     public function __construct(
+        private readonly ConfigProvider $configProvider,
         ?ClientInterface $client = null,
     ) {
         $this->client = $client ?? new GuzzleClient(
             [
-                'base_uri' => ConfigProvider::BASE_URL,
+                'base_uri' => $this->configProvider->baseUrl,
```

Comment:
API version can be included in base url? Since we have base url in env, we can change api version dynamically. 
What I mean is removing the API version env variable.

Reply:
In my opinion, it is better to keep the version as a variable and not put it in base_url

Reply:
Any specific reason? :D 
Take a look at the variables in env that end with `_URL`

Reply:
I agree with @olekans

Reply:
Changed :)